_posts/2020-08-09-actual-advice-1.md

```diff
@@ -0,0 +1,15 @@
+This is the first blog post of a format I'd like to use to share advice I received throughout my career. There is so much advice out there that in most cases, you'll always find someone defending and someone else challenging any type of advice, so I won't claim this advice to be universally correct. I'd like to share the advice that worked for me personally, that made me better, not necessarily only at my job.
+
+I received this first advice from my highly valued colleague and co-worker Daniel Marbach. While he is a genius software engineer, he is also a gym enthusiast. Some day, I expressed my frustration about the lack of enjoyment and motivation I get from doing fitness exercises. I've mostly tried bodyweight training as I could do my exercises at home but I failed at completing exercise goals several times. I struggled to find motivation, causing me to find excuses to skip training again and again. At some point, you just stop completely because you give up your hope of achieving your training plan. Daniel's advise was very simple and you probably wonder why I even bother to write this down: Set yourself a goal that you are confident to achieve every single time. For me, this goal was to do my exercises just once a week.
+
+Admittedly, this goal is not very ambitious and every fitness trainer would probably also tell me how inefficient this is but this is not what this small goal is about. The goal should help you to build up a new habit. And for this, it workes like a charm. At the time of writing this, I did my exercises once every single week for about 6 months. This is far beyond the timeframe I managed to keep up my exercise schedule before and the habit of doing my exercises became more and more natural. Doing your exercises once a week consistently over many months or years is still better than doing it three times a week for only a month. 
+
+This won't make me the next Arnold Schwarzenegger so you might say this is bad advice for people looking to improve their career or business in an impactful way. But on the contrary, this approach of setting yourself achievable goals and getting into the habit of meeting them might be exactly what you need. Once you have identified what you want to improve at, start with small goals, start with goals that you can achieve. Setting absurdly high goals that you can't meet almost all of the time will make you start ignoring those. You'll constantly miss your ambitious goals and you'll start not bothering anymore. You'll probably even become frustrated and give up. Achievable goals will help you to stay motivated, they will help you to build up your skills reliably.
+
+I still don't like exercising but I keep doing it and I keep improving so it becomes more natural every single time. When defining your goal, accept the fact that your life is busy and that it probably will be easier to meet a weekly goal instead of a daily goal. Don't worry about starting slow, maintain the pace that keeps you going.
```

Comment:
```suggestion
Still, I wouldn't say I like exercising, but I keep doing it, and I keep improving, so it becomes more natural every single time. When defining your goal, accept the fact that your life is busy and that it probably will be easier to meet a weekly goal instead of a daily goal. Don't worry about starting slow; maintain the pace that keeps you going.
```